hunts/migrations/0009_huntsettings_google_sheets_template_folder_id_and_more.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.0.1 on 2022-01-13 05:29
+
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ("hunts", "0008_huntsettings_backfill"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="huntsettings",
+            name="google_sheets_template_folder_id",
+            field=models.CharField(blank=True, max_length=128),
+        ),
+        migrations.AlterField(
```

Comment:
Oops this is my bad, I changed this earlier but never generated a migration for it. Looks like it's also included in #573